gr-digital/lib/ofdm_carrier_allocator_cvc_impl.h

```diff
@@ -1,19 +1,19 @@
 /* -*- c++ -*- */
-/* 
- * Copyright 2013 Free Software Foundation, Inc.
- * 
+/*
+ * Copyright 2013-2018 Free Software Foundation, Inc.
```

Comment:
I suppose this should be `Copyright 2013, 2018`.
Also, there seems to be no code changes in this file. Why update the copyright notice then?